server/src/main/java/org/elasticsearch/action/ActionModule.java

```diff
@@ -586,7 +586,7 @@ public ActionModule(
     public void copyRequestHeadersToThreadContext(HttpPreRequest request, ThreadContext threadContext) {
         // the request's thread-context must always be populated (by calling this method) before undergoing any request related processing
         // we use this opportunity to first record the request processing start time
-        threadContext.putTransient(Task.TRACE_START_TIME, Instant.ofEpochMilli(threadPool.absoluteTimeInMillis()));
+        threadContext.putTransient(Task.TRACE_START_TIME, Instant.ofEpochMilli(System.currentTimeMillis()));
```

Comment:
The one downside of this is testability - don't we have a `Clock` that we can inject? But maybe that's overkill and this is good enough.

Reply:
Eh we probably could (indeed we could make a `ThreadPool#rawAbsoluteTimeInMillis` to parallel `ThreadPool#rawRelativeTimeInMillis`). Not sure it's really worth it here, at least not unless we see this test actually failing.